src/components/shared/chart/Canvas.js

```diff
@@ -269,11 +277,20 @@ export class ChartCanvas<HoveredItem> extends React.Component<
 
     const maybeHoveredItem = this.props.hitTest(this._offsetX, this._offsetY);
     if (maybeHoveredItem !== null) {
-      this.setState({
-        hoveredItem: maybeHoveredItem,
-        pageX: event.pageX,
-        pageY: event.pageY,
-      });
+      if (this.state.selectedItem === null) {
+        // Update both the hovered item and the position of the tooltip.
+        this.setState({
+          hoveredItem: maybeHoveredItem,
+          pageX: event.pageX,
+          pageY: event.pageY,
+        });
+      } else {
+        // If there is a selected item, only update the hoveredItem and not the
+        // pageX and pageY values which is used for the position of the tooltip.
```

Comment:
same here: _why_ are we not updating these values in this case?

Reply:
actually it's not clear to me why we change hoveredItem when there's a selected item, despite that we won't do anything with it. Note that this will trigger a rerender. I'd like to have more information about that.

Reply:
If you click on a marker and hover over another marker afterwards, we still want to be able to add a hover effect (color change) to the hovered marker. This was the behavior before this patch as well so I decided to keep it consistent. I think it makes sense to see which marker you are hovering.